src/main/java/frc/robot/Constants.java

```diff
@@ -113,4 +115,22 @@ private Positions(Distance position) {
             }
         }
     }  
+
+    public static class IntakeK {
+        // IDs
+        public static final int motorLeftId = 0; //! find
+        public static final int motorRightId = 1; //! find
+        public static final int timeOfFlightId = 2; //! find
```

Comment:
```suggestion
        public static final int timeOfFlightId = 0;
```
This is has a different can device type so it won't conflict with the motors